mobile-app/cypress/e2e/functional/wallet/authorization/transactionAuthorization.spec.ts

```diff
@@ -40,7 +50,7 @@ context('Wallet - Transaction Authorization with Error', () => {
     })
     cy.getByTestID('cancel_authorization').click()
     cy.getByTestID('button_confirm_convert').click().wait(2000)
-    cy.getByTestID('pin_attempt_error').should('not.exist')
+    cy.getByTestID('pin_attempt_error').should('exist') //  error message should exist when incorrect pin is entered.
```

Comment:
Did we change this behavior? I think the logic here is, when cancelled, it shouldn't show any error message.

Reply:
When canceled it will still show the attemps remaining. That's the behavior right now